src/tavi/frontend/presenters/file_menu_presenter.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from tavi.frontend.views.file_menu_view import FileMenu
+
+if TYPE_CHECKING:
+    from tavi.backend.model_interface import TaviProjectInterface
+
+
+class FileMenuPresenter:
+    """
+    Presenter responsible for mediating interactions between the main menu bar
+    (`MainMenuBar`) and the project model (`TaviProjectInterface`).
+    Parameters
+    ----------
+    view : MainMenuBar
+        The menu bar view that contains user-triggered actions (e.g., load folder,
+        load files, new project, save, exit).
+    model : TaviProjectInterface
+        The underlying project model that handles loading, saving, and maintaining
+        TAVI project state.
+    """
+
+    def __init__(self, exit_routine, model: TaviProjectInterface):
+        super().__init__()
+        self._view = FileMenu()
+        self.exit_routine = exit_routine
```

Comment:
why does this attribute not begin with an underscore `_`?  I think this is local and private to the class.

Reply:
added _